pkg/profiler/cpu/cpu.go

```diff
@@ -388,6 +410,18 @@ func (p *CPU) report(lastError error, processLastErrors map[int]error) {
 	p.processLastErrors = processLastErrors
 }
 
+func (p *CPU) addUnwindTables(pid int) error {
```

Comment:
```suggestion
func (p *CPU) loadUnwindTables(pid int) error {
```
Load the unwind tables to the kernel? Or sync the unwind tables to the kernel?

Reply:
I would prefer loading or adding, as the BPF map is there and we are adding / loading/inserting values. They are like a map in any other runtime or language, so I think "syncing" makes it sound like we are doing something different, but not too attached to this naming